page_object_ecomap/framework/Pages.py

```diff
@@ -250,6 +290,10 @@ def is_success_popup_present(self):
             return False
         return True
 
+    def get_popup_text(self):
+        return self.find_element(*ProblemLocator.POP_UP_WINDOW_TITLE).text
```

Comment:
In my opinion, u can define separate method get_text in BasePage and invoke it here, instead of using find_element.text